apps/challenges/challenge_config_utils.py

```diff
@@ -0,0 +1,179 @@
+import requests
+import zipfile
+import yaml
+
+from os.path import basename, isfile, join
+from rest_framework import status
+
+
+def write_file(output_path, mode, file_content):
+    with open(output_path, mode) as file:
+        file.write(file_content)
+
+
+def extract_zip_file(file_path, mode, output_path):
+    zip_ref = zipfile.ZipFile(file_path, mode)
+    zip_ref.extractall(output_path)
+    zip_ref.close()
+    return zip_ref
+
+
+def get_yaml_files_from_challenge_config(zip_ref):
+    """
+        Arguments:
+            zip_ref {zipfile} -- reference to challenge config zip
```

Comment:
Let's be precise with the docstring. I think it is the path to the zip file.

Reply:
no it's the actual zipfile reference